lib/run-script-pkg.js

```diff
@@ -59,10 +45,17 @@ const runScriptPkg = async options => {
     return { code: 0, signal: null }
   }
 
-  if (stdio === 'inherit' && banner !== false) {
-    // we're dumping to the parent's stdout, so print the banner
-    console.log(bruce(pkg._id, event, cmd, args))
+  let banner
```

Comment:
I think we still might want to guard this with `if (stdio === 'inherit')`.

For example, we set `stdio: pipe` if `--foreground-scripts=false` is set. In that case we will still get `output` events in the display layer but we'll need to know if they came from a child process so we can use `config.foregroundScripts` to hide them.